2022/24xxx/CVE-2022-24727.json

```diff
@@ -2,7 +2,7 @@
     "CVE_data_meta": {
```

Comment:
{} is not allowed for {'problemtype_data': [{'description': [{'lang': 'eng', 'value': "CWE-77: Improper Neutralization of Special Elements used in a Command ('Command Injection')"}]}]}